apps_product_creator/models/odoo_module.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2017-Today: Odoo Community Association (OCA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import models, fields, api
+
+
+class OdooModule(models.Model):
+    _inherit = 'odoo.module'
+
+    @api.depends(
+        'product_template_id',
+        'product_template_id.product_variant_ids'
+    )
+    def _get_rel_product_count(self):
+        for module in self:
+            module.rel_product_count = len(
+                module.product_template_id.product_variant_ids)
+
+    product_template_id = fields.Many2one(
+        'product.template',
+        "Product Template",
+    )
+    rel_product_count = fields.Integer(
```

Comment:
product_qty is maybe more understandable. (rel is not very used in the name of fields).